src/components/Text/__snapshots__/Text.test.tsx.snap

```diff
@@ -0,0 +1,79 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I thought snapshots were being skipped 🤔

Reply:
Oh were they for `Text` or in general?